opy/_regtest/src/_devbuild/gen/types_asdl.py

```diff
@@ -14,6 +14,7 @@ class bool_arg_type_e(py_meta.SimpleObj):
 class redir_arg_type_e(py_meta.SimpleObj):
   ASDL_TYPE = TYPE_LOOKUP.ByTypeName('redir_arg_type')
 
+redir_arg_type_e.Unknown = redir_arg_type_e(0, 'Unknown')
```

Comment:
Oh please revert these two opy/_regtest files ... That is an old snapshot of the code that doesn't need to be kept up to date.  It's only for testing the bytecode compiler (which we won't need after translation is done)